lib/xcake/generator/project.rb

```diff
@@ -25,6 +25,9 @@ def visit_project(project)
 
         @project = Xcode::Project.new(output_filepath, true)
         @project.setup_for_xcake
+
+        @project.attributes['CLASSPREFIX'] = project.class_prefix if project.class_prefix
```

Comment:
Line is too long. [89/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.